src/mocker_node/src/node/mocker_node.cpp

```diff
@@ -2,23 +2,25 @@
 
 #include <filesystem>
 
-MockerNode::MockerNode() : rclcpp::Node("mocker_node") {
+MockerNode::MockerNode(const std::string &track_name, const std::string &sim) : rclcpp::Node("mocker_node") {
   std::string planning_gtruth_default_file_path =
       std::string(std::filesystem::current_path()) +
-      "/gtruths/tracks_gtruths/track1/track1_gtruth.csv";
+      "/gtruths/tracks/" +  sim + "/" + track_name + "/" + track_name + "_gtruth.csv";
   std::string se_gtruth_default_file_path =
-      std::string(std::filesystem::current_path()) + "/gtruths/tracks_gtruths/track1/track1.csv";
+      std::string(std::filesystem::current_path()) + "/gtruths/tracks/" + sim + "/" + track_name + "/" + track_name + ".csv";
+
   std::string planning_gtruth_file = declare_parameter<std::string>(
       "planning_gtruth_file_path", planning_gtruth_default_file_path);
   std::string se_gtruth_file =
       declare_parameter<std::string>("se_gtruth_file_path", se_gtruth_default_file_path);
+      
   gtruth_planning = planning_gtruth_fromfile(open_file_as_stream(planning_gtruth_file));
   gtruth_se = se_gtruth_fromfile(open_file_as_stream(se_gtruth_file));
 
   planning_publisher = this->create_publisher<custom_interfaces::msg::PathPointArray>(
       "planning_gtruth", rclcpp::QoS(10).durability(RMW_QOS_POLICY_DURABILITY_TRANSIENT_LOCAL));
   se_publisher = this->create_publisher<custom_interfaces::msg::ConeArray>(
-      "track_map", rclcpp::QoS(10).durability(RMW_QOS_POLICY_DURABILITY_TRANSIENT_LOCAL));
+      "/state_estimation/map", rclcpp::QoS(10).durability(RMW_QOS_POLICY_DURABILITY_TRANSIENT_LOCAL));
```

Comment:
But planning needs to read from this topic, so it needs to be the same one used in state estimation

Reply:
@marhcouto

Reply:
If it came to that we could just pass a parameter to planning for the name of the topic. But we had agreed that this could stay here, even though we were not going to use it, so in practice it won't

Reply:
I didn't understand the final conclusion. It stays or we change slightly?